packages/core/src/messages/responses/cache-sorted-set-get-score.ts

```diff
@@ -26,7 +26,14 @@ import {SdkError} from '../../errors';
  * }
  * ```
  */
-export abstract class Response extends ResponseBase {}
+export abstract class Response extends ResponseBase {
+  public score(): number | undefined {
```

Comment:
I understand the reasoning, but it feels a little off for this and the increment response to be the only ones without a value() method. Would it be worth having both, or do we not want to confuse users since the 'value' of the set is never returned?

Reply:
i'm just choosing to name them the same thing that they are already named on the `Hit` class. I think it would be weird for the parent class to have a different name than the accessor on the `Hit` class?

Reply:
Fair. I was picturing a hypothetical generic Response interface with a value() method. I suppose the question then would be whether `Hit` should have value(). Just a thought, especially since adding it would bring this out of step with the other SDKs.

Reply:
yeah I see what you're getting at for sure, but I think what makes the most sense to me is just a pattern that lets the user pretend like the parent class is the corresponding `Hit` or `Success` subclass.

Reply:
I am good with score. I share the same hesitation as Nate, but since this is a `get score` api, i'm ok with this